### PR TITLE
Additions due to new features for dbx extension

### PR DESCRIPTION
**Pull request by chaitya-patel**
Added additional step and updated for new features

### CODE REVIEW

File: tutorials/hana-dbx-extension/hana-dbx-extension.md [hunk 1]
```diff
@@ -144,14 +144,25 @@ The SAP HANA database explorer extension also allows you to connect to the datab
...
+
+    ![Open Settings](dbx-settings.png)
+
+    - **Max Sql Result Size**: Here you can change the maximum amount of SQL results that you want to allow to be returned by any queries that you run. 
+
+    - **Show Database Explorer Connections**: By selecting/deselecting this checkbox, you can show/hide the SAP HANA Database Explorer connections in your Database List. Note that this will not affect your local connections that are only available in Visual Studio Code.
+
+    ![DBX Settings](dbx-settings2.png)
+
+3. Visual Studio Code also has built-in functionality that allows you to split editors which can be very useful to customize your workspace. For example, if you are working on multiple tasks, or need to refer to a table while manipulating another one, this feature can allow you to work more efficiently. In a SQL Console, run the following code.
```
Comment (MichaelCzcz) [inline tutorials/hana-dbx-extension/hana-dbx-extension.md:165]: Missing comma before which

File: tutorials/hana-dbx-extension/hana-dbx-extension.md [hunk 2]
```diff
@@ -183,7 +194,7 @@ Learn how to change appearance settings and leverage features such as split edit
 
     ![Compare Tables](compareTables.png)
 
-3. Additionally, Visual Studio Code has other features to help organize your workspace. The icons in the top right corners allow you to toggle (hide/show) different parts of the workspace which you can also do by resizing and snapping different areas. Click on the the second icon to toggle the panel and use the Terminal to create a new file for your SQL code.
+4. Additionally, Visual Studio Code has other features to help organize your workspace. The icons in the top right corners allow you to toggle (hide/show) different parts of the workspace which you can also do by resizing and snapping different areas. Click on the the second icon to toggle the panel and use the Terminal to create a new file for your SQL code.
```
Comment (MichaelCzcz) [inline tutorials/hana-dbx-extension/hana-dbx-extension.md:197]: Missing comma before which

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/hana-dbx-extension/hana-dbx-extension.md (hunk 2 above)
Comment (chaitya-patel) [reply tutorials/hana-dbx-extension/hana-dbx-extension.md]: done!

File: tutorials/hana-dbx-extension/hana-dbx-extension.md (hunk 1 above)
Comment (chaitya-patel) [reply tutorials/hana-dbx-extension/hana-dbx-extension.md]: done!

**Pull request merged**